backend/src/main/java/com/bikes/backend/model/MongoUserDTO.java

```diff
@@ -0,0 +1,4 @@
+package com.bikes.backend.model;
+
+public record MongoUserDTO(String username, String password) {
```

Comment:
DTO ist to generic. Use a more specifig name like MongoUserRequest
```suggestion
public record MongoUserRequest(String username, String password) {
```